### ISSUE TITLE
[Code] Prompt mode.

### ISSUE THREAD

**Issue by Symbiomatrix** [hako-mikan/sd-webui-regional-prompter#111]
```
                    if self.ex:
                        ox = torch.where(masked !=0 , masked, ox)
                    else:
                        ox = ox + masked
                    maskb = maskb + mask if maskb is not None else mask
                maskb = 1 - maskb
                if not self.ex : ox = ox + torch.multiply(outb, maskb)
```
@hako-mikan I see that when multiple prompt regions overlap, maskb will receive the cumulative count, and then negate the base accordingly. Wouldn't it be more sensible to average out the effect of each mask? For example, if first mask has a difference of 2 and second mask has a difference of 1, then instead of 1.5 along the intersection you get 3. I might be misreading, but that's what it looks like.

Also, if the token is not present in the output, does it apply to the entire image? Because I see in the output that when this happens the mask is entirely white, which would otherwise indicate full application. Might just be a display issue though.

**Comment by hako-mikan** [comment]
I thought about this when implementing it. Of course, it seems better to average it. However, I was concerned that averaging would make the effect ambiguous. In attention mode, even if there is some overlap, the image does not collapse, so it is set up this way. However, I planned to make adjustments if there were any issues.

**Comment by Symbiomatrix** [comment]
Alright, thanks. I'll run a comparison later to see whether it's a useful difference.
